sky/tools/create_full_ios_framework.py

```diff
@@ -0,0 +1,214 @@
+#!/usr/bin/env python3
+#
+# Copyright 2013 The Flutter Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+# Generates and zip the ios flutter framework including the architecture
+# dependent snapshot.
+
+import argparse
+import subprocess
+import shutil
+import sys
+import os
+
+from create_xcframework import create_xcframework
+
+DSYMUTIL = os.path.join(
+    os.path.dirname(__file__), '..', '..', '..', 'buildtools', 'mac-x64',
+    'clang', 'bin', 'dsymutil'
+)
+
+buildroot_dir = os.path.abspath(
+    os.path.join(os.path.realpath(__file__), '..', '..', '..', '..')
+)
+
+
+def main():
+  parser = argparse.ArgumentParser(
+      description=(
+          'Creates Flutter.framework, Flutter.xcframework and '
+          'copies architecture-dependent gen_snapshot binaries to output dir'
+      )
+  )
+
+  parser.add_argument('--dst', type=str, required=True)
+  parser.add_argument('--clang-dir', type=str, default='clang_x64')
+  parser.add_argument('--x64-out-dir', type=str)
+  parser.add_argument('--arm64-out-dir', type=str, required=True)
+  parser.add_argument('--armv7-out-dir', type=str, required=False)
```

Comment:
Nothing should be using this arg anymore as of https://flutter-review.googlesource.com/c/recipes/+/29060, all `armv7` logic should be removed.  I guess we missed that cleanup in 
https://github.com/flutter/engine/blob/81f540020ad0a0e129f4b8d026ceeff45dcb4fd0/sky/tools/create_macos_gen_snapshots.py#L37
```suggestion
```

Reply:
Argument is still here, are we calling it somewhere?